pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -347,18 +349,14 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(cniTimeoutCtx, plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
-	return fmt.Sprintf("%s_%s/%s", namespace, name, id.ID)
```

Comment:
`format.PodDesc` is using `"%s_%s(%s)"` as a format. Here the `/` is used. It may be intentional.

Reply:
Comment to when this method was introduced (for history): https://github.com/kubernetes/kubernetes/pull/69159#discussion_r221402772

Reply:
and method existed long before this: https://github.com/kubernetes/kubernetes/pull/36020/files